modules/es-extensions/store/extensions/app/greg_store/pages/greg-landing.jag

```diff
@@ -22,7 +22,9 @@ require('/modules/store.js').exec(function(ctx) {
 
 	var log = new Log();
 	log.debug("Printing user ############" + stringify(user));
-	var tenantId = user.tenantId;
+    
+    var tenantDetails = tenantApi.createTenantDetails(request, session);
```

Comment:
Fixed in https://github.com/wso2/product-greg/pull/856/commits/9bf8e288c7bfad60a83fff703e0368b94d78f5ae